<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DayNightScheAdd</class>
 <widget class="QWidget" name="DayNightScheAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>Schedule Add</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_background" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_startTimeAdd">
          <property name="text">
           <string>Start Time</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_endTimeAdd">
          <property name="text">
           <string>End Time</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_exposureLevelAdd">
          <property name="text">
           <string>Exposure Level</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ComboBox" name="comboBox_exposureLevelAdd"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_minShutterAdd">
          <property name="text">
           <string>Minimum Shutter</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="ComboBox" name="comboBox_minShutterAdd"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_maxShutterAdd">
          <property name="text">
           <string>Maximum Shutter</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="ComboBox" name="comboBox_maxShutterAdd"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_gainLevelAdd">
          <property name="text">
           <string>Limit Gain Level</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="MyLineEdit" name="lineEdit_gainLevelAdd"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_latencyIrCutAdd">
          <property name="text">
           <string>IR-CUT Latency</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="ComboBox" name="comboBox_latemcyIrCutAdd"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_irCutAdd">
          <property name="text">
           <string>IR-CUT</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="ComboBox" name="comboBox_irCutAdd"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_irLedAdd">
          <property name="text">
           <string>IR LED</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="ComboBox" name="comboBox_irLedAdd"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_colorModeAdd">
          <property name="text">
           <string>Color Mode</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="ComboBox" name="comboBox_colorModeAdd"/>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="MySpinBox" name="spinBox_startHour"/>
          </item>
          <item>
           <widget class="MySpinBox" name="spinBox_startMinute"/>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="MySpinBox" name="spinBox_endHour"/>
          </item>
          <item>
           <widget class="MySpinBox" name="spinBox_endMinute"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ok_add">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_cancel_add">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>MySpinBox</class>
   <extends>QSpinBox</extends>
   <header>myspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
